site/content/includes/m3query/annotated_config.yaml

```diff
@@ -0,0 +1,246 @@
+# The server listen address
+listenAddress: <url>
+
+# Metrics configuration
+# TODO: Which is what?
+metrics:
+  # Scope of metrics root
+  # TODO: Again, which is?
+  scope:
+    # Prefix prepended to metrics collected
+    prefix: <string>
+    # Reporting frequendy of metrics collected
+    reportingInterval: <duration>
+    # Tags shared by metrics collected
+    tags: <map of strings>
+  # Configuration for a Prometheus reporter (if used)
+  prometheus:
+    # Metrics collection endpoint for application
+    # Default = "/metrics"
+    handlerPath: <string>
+    # Listen address for metrics
+    # Default = "0.0.0.0:7203"
+    listenAddress: <url>
+    # Metric sanitization type, valid options: [none, m3, prometheus]
+    # Default = "none"
+    sanitization: <string>
+  # Sampling rate for metrics. min=0.0, max=1.0
+  # TODO: What does this mean exactly?
+  samplingRate: <float>
+  # Enable Go runtime metrics, valid options: [none, simple, moderate, detailed]
+  # See https://github.com/m3db/m3/blob/master/src/x/instrument/extended.go#L39:L64 for more details
+  extended: <string>
+
+# Logging configuration
+# TODO: More detail than this 
+# https://github.com/m3db/m3/blob/9f129cf9f16430cc5a399f60aa5684fb72b55bb5/src/cmd/services/m3query/config/config.go#L116
+logging:
+  level: info
+
+# Enables tracing, if nothing configured, tracing is disabled
+tracing:
+  # Name for tracing service
+  serviceName: <string>
+  # Tracing backen to use, valid options: [jaeger, lightstep]
+  backend: <string>
+  # If using Jaeger, options to send to tracing backend
+  jaeger:
+  # If using Lightstep, options to send to tracing backend
+  lightstep:
+
+clusters:
+  - namespaces:
```

Comment:
The namespaces section can be removed here.

Reply:
I'll save that for the other in progress PR, but thanks for the note :)